components/blitz/src/omero/cmd/basic/DoAllI.java

```diff
@@ -259,12 +259,7 @@ public void init(Helper helper) {
             Map<String, String> allgroups = new HashMap<String, String>();
             allgroups.put("omero.group", "-1");
             ctx.publishMessage(new ContextMessage.Push(this, allgroups));
-            try {
-                // Process within -1 block.
-                GraphUtil.combineFacadeRequests(this.requests);
-            } finally {
-                ctx.publishMessage(new ContextMessage.Pop(this, allgroups));
-            }
+            ctx.publishMessage(new ContextMessage.Pop(this, allgroups));
```

Comment:
I have no idea: I don't know what they do nor who/what's listening for those (ha, that's often an issue with Insight) so I figure they must have been carried across somehow from older code. Happy to try removing them if you like!